app/routes/public-services.js

```diff
@@ -11,12 +11,16 @@ import LpdcInputComponent from 'frontend-lpdc/components/rdf-form-fields/lpdc-in
 import LpdcDateTimeComponent from 'frontend-lpdc/components/rdf-form-fields/lpdc-date-time';
 import LpdcRdfInputFieldsConceptSchemeMultiSelectorComponent from 'frontend-lpdc/components/rdf-form-fields/lpdc-concept-scheme-multi-selector';
 import LpdcRdfHeadingComponent from 'frontend-lpdc/components/rdf-form-fields/lpdc-heading';
+import { HttpRequest } from 'frontend-lpdc/helpers/http-request';
```

Comment:
Out of scope for this PR, something that could be improved in a future PR if time permits:

I would move the http-request to a different folder. "helper" has an existing meaning in Ember apps and this is clearly not a helper in the traditional sense :sweat_smile:. It confused me when I read that line.